components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/WorkFlowExecutorManager.java

```diff
@@ -118,17 +120,18 @@ public WorkflowExecutorResult executeWorkflow(WorkflowRequest workFlowRequest) t
                     List<Parameter> parameterList = workflowDAO.getWorkflowParams(association.getWorkflowId());
                     templateImplementation.execute(requestToSend, parameterList);
                     workflowRequestAssociationDAO.addNewRelationship(relationshipId, association.getWorkflowId(),
-                                                                     workFlowRequest
-                                                                             .getUuid(), WorkflowRequestStatus.PENDING
-                                                                             .toString(), workFlowRequest.getTenantId());
+                            workFlowRequest
+                                    .getUuid(), WorkflowRequestStatus.PENDING
+                                    .toString(), workFlowRequest.getTenantId());
                 }
             } catch (JaxenException e) {
-                String errorMsg = "Error when executing the xpath expression:" + association.getAssociationCondition() + " , on " +
-                                  xmlRequest ;
-                log.error( errorMsg, e);
+                String errorMsg = "Error when executing the xpath expression:" + association.getAssociationCondition
+                        () + " , on " +
+                        xmlRequest;
```

Comment:
Thanks for pointing, fixed with https://github.com/wso2/carbon-identity/pull/1675